### PR TITLE
Removed Hardcoded Texts

### LINKED ISSUE #223: Remove hardcoded texts from XML files
**Summary:**
Using hardcoded texts in XML files is not a good practice. We need to add them in string resources and use them. By inspecting the project I found there are 93 hardcoded texts in the whole project. I am interested in adding them in string resources.

**Screenshot:**
screenshot from 2019-01-19 01-59-19

### PR DESCRIPTION
**Pull request by RajaVamsi11**
**Summary:**
Fixes #223 
Removed all hardcoded texts from XML files and placed them in the strings.xml file.

**Screenshots:**
**Before Removing:**
screenshot from 2019-01-19 01-59-19

**After Removing:**
screenshot from 2019-01-19 02-25-41

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@luckyman20 Please review.

File: mifospay/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -121,6 +114,81 @@
...
+    <string name="mm">MM</string>
+    <string name="slash">/</string>
+    <string name="search">Search</string>
+    <string name="enter_otp">Enter OTP</string>
+    <string name="vpa">VPA</string>
+    <string name="enter_upi_pin">Enter UPI PIN</string>
+    <string name="history">History</string>
+    <string name="notifications">Notifications</string>
+    <string name="bank_name">Bank Name</string>
+    <string name="a_c_holder_name">A/C Holder Name</string>
```
Comment (luckyman20) [inline mifospay/src/main/res/values/strings.xml]: Please change it to `ac_holder_name`

**Comment by RajaVamsi11** [comment]
@luckyman20 Done.

**Comment by RajaVamsi11** [comment]
@ankurs287 Can you please review this?

**Pull request merged**